### PR TITLE
dataset + param changes + sfm

### PR DESCRIPTION
**Pull request by roddylan**

### CODE REVIEW

File: sfm/reconstruction.py [hunk 1]
```diff
@@ -97,21 +97,22 @@ def poisson(self, depth=8, width=0, scale=1.25, l_fit=False, pcd=None):
...
             pcd = self.pcd
         
-        if radius is None:
+        if radii is None:
             D = pcd.compute_nearest_neighbor_distance()
             avg_D = np.mean(D)
             radius = 3 * avg_D
+            radii = [radius, radius * 2]
         
         radius = np.float64(radius)
```
Comment (akhadeli) [inline sfm/reconstruction.py]: this line is no longer necessary

**Review by akhadeli — CHANGES_REQUESTED** [review]

File: sfm/reconstruction.py (hunk 1 above)
Comment (akhadeli) [reply sfm/reconstruction.py]: im talking about "radius = np.float64(radius)"

Comment (roddylan) [reply sfm/reconstruction.py]: o word good now?

Comment (akhadeli) [reply sfm/reconstruction.py]: looks good

**Review by akhadeli — APPROVED** [review]

**Pull request merged**